test/EndToEndSpec.hs

```diff
@@ -1825,24 +1861,38 @@ library #{name}
   default-language: Haskell2010
 |]
 
+-- As for executable_, but without the final "  default-language: Haskell2020"
+executable'_ :: String -> String -> Package
```

Comment:
This is a difficult decision. On the one hand we want to keep things DRY (like you did 👍), on the other hand we don't want too many versions of ` executable*` and `library*`. We can't have both.

In this particular case I'm in favor of the duplication. I'll take care of it.